### ISSUE TITLE
Heatmap for story pictures

### ISSUE THREAD

**Issue by samuel-git** [weglide/bugtracker#117]
Sometimes you are looking for pictures of a special location.
It would be an amazing feature to show **all** story pictures on one map.
Since thousands of pictures would flood the map, it would be an idea to sum up the pictures to a heatmap.
When zooming in, the heatmap gets more detailed.

As an example I added screenshots of my google photos account:
![heatmap](https://user-images.githubusercontent.com/47079209/112160366-60593800-8bea-11eb-90d2-aef229812c6b.jpg)

Multiple heatmaps could be provided (all pictures, my club pictures, my own pictures).

_Originally posted by @frantzheld in https://github.com/weglide/bugtracker/issues/80#issuecomment-804937853_

**Comment by moldhouse** [comment]
@samuel-git  what would the endpoint need to look like? I guess for your own pictures and club pictures we could do a simple geojson file, but for all pictures, there would need to be some kind of tiles?

**Comment by samuel-git** [comment]
Yes, simple point feature collection as geojson or tiles would be perfect. I would use clustered points instead of heatmaps because they are interactive.

**Comment by samuel-git** [comment]
My suggestion are clusters like these:
![image](https://user-images.githubusercontent.com/22152728/124894365-ddfb5500-dfdb-11eb-86b9-770532a3d106.png)
Makes clicking a photo icon easy.

Maybe merge the map with https://www.weglide.org/story

**Comment by samuel-git** [comment]
Komoot just introduced this feature, called "trail view":

<img width="668" alt="grafik" src="https://user-images.githubusercontent.com/22152728/188280671-aa37a470-3e20-421e-a06a-e748df7e53e6.png">

**Comment by deltafox44** [comment]
Such a feature would help solve [Showing ALL flights in a given area](https://github.com/weglide/bugtracker/issues/187#top)... especially is there were the same type of heatmap for ascendances - each spot with a positive vario over a given period of time - 30 seconds or so
